### DISCUSSION TITLE
Website dark theme

### DISCUSSION THREAD

**Discussion by KaKi87** [phosphor-icons/homepage#461] [Ideas]
Hi,

It would be nice to have a dark theme for the website.

Thanks

**Comment by KaKi87** [comment]
Hi @rektdeckard, would it be possible to have some feedback about this request ? Thanks

**Reply by rektdeckard** [reply]
We'll look into it, but given how much of the above-the-fold region is graphical assets, its more than just picking a few colors. In the mean time, you can bookmark https://phosphoricons.com/?color=%22ffffff%22#toolbar which will take you right to the icon grid with a light color (and thus dark background) set.

**Reply by KaKi87** [reply]
The workaround is fine for the main section, but it doesn't apply to the header and footer sections, which are actually about picking a few colors. Thanks

**Reply by rektdeckard** [reply]
For normal people, it is probably as easy as picking a few colors. For design-obsessed people like us, we will redesign an entirely different appearance for dark mode 😆

**Reply by KaKi87** [reply]
Well, then, maybe you'll get motivated by seeing how people like me make your site look like by using [Dark Reader](https://github.com/darkreader/darkreader) :sweat_smile: 

![](https://github.com/phosphor-icons/homepage/assets/21284089/6a0bcd98-2f51-4c91-b4dd-84f500ce438f)

**Reply by rektdeckard** [reply]
😱

**Reply by rektdeckard** [reply]
What's up with the color picker? I'm guessing this is Firefox?

**Reply by KaKi87** [reply]
Nope, it's Chromium, but the extension is cross-browser so the effect should be the same.
